Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Light/LightBehaviorXaml.bind

```diff
@@ -0,0 +1,32 @@
+<Page
```

Comment:
Ok I may have an issue locally..anyhow we'll check it next week if it is not the case